File: micronaut-java-parent-example/rest/src/test/java/io/archimedesfw/maven/micronaut/example/todo/ApplicationIT.java
package io.archimedesfw.maven.micronaut.example.todo;

import io.micronaut.runtime.EmbeddedApplication;
import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
import jakarta.inject.Inject;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

@MicronautTest
class ApplicationIT {

    @Inject
    private EmbeddedApplication<?> application;

    @Test
    void bootstrap() {
        assertTrue(application.isRunning());
    }

}

File: micronaut-kotlin-parent-example/rest/src/test/kotlin/io/archimedesfw/maven/micronaut/example/todo/ApplicationIT.kt
package io.archimedesfw.maven.micronaut.example.todo

import io.micronaut.runtime.EmbeddedApplication
import io.micronaut.test.extensions.junit5.annotation.MicronautTest
import jakarta.inject.Inject
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

@MicronautTest
internal class ApplicationIT {

    @Inject
    lateinit var application: EmbeddedApplication<*>

    @Test
    fun bootstrap() {
        assertTrue(application.isRunning)
    }

}
